app/js/daemonManager.js

```diff
@@ -63,21 +64,21 @@ function DaemonManager() {
 
 	/**
 	 * Checks if there is an update available
+	 * @function DaemonManager#update
 	 */
 	function updatePrompt() {
-		if (!self.Running) {
-			UI.notify('siad is not running!', 'stop');
-			return;
-		}
 		apiCall("/daemon/updates/check", function(err, update) {
 			if (err) {
-				UI.notify('Update check failed!', 'error');
-				return;
+				self.Running = false;
+				// Check again later
+				setTimeout(updatePrompt, 1000);
```

Comment:
Makes it so update check will delay API calls until successful. Won't return "Update check failed!" and will wait the duration that it takes to load up the blockchain. That's desired behavior, right?